ssi-sd-jwt/src/serialized.rs

```diff
@@ -0,0 +1,121 @@
+/// Lower level API to encode a fully encoded SD-JWT given a JWT and disclosure array
+/// already fully encoded into their string representations
+pub fn serialize_string_format(jwt: &str, disclosures: &[&str]) -> String {
+    let mut serialized = format!("{}~", jwt);
+
+    for disclosure in disclosures {
+        serialized = format!("{}{}~", serialized, disclosure)
+    }
+
+    serialized
+}
+
+#[derive(Debug, PartialEq)]
+pub struct Deserialized<'a> {
+    pub jwt: &'a str,
+    pub disclosures: Vec<&'a str>,
```

Comment:
We don't want to because we need to maintain the string format at this stage.  The lack of canonicalization means that the specific byte encoding represented by the base64 is significant for the hashes being processed downstream.  Additionally at this stage we don't know the hash algorithm in question in order to generate the hash.